shipstation_integration/hooks.py

```diff
@@ -61,8 +61,7 @@
 # Installation
 # ------------
 
-# TODO: move to custom JSON: https://github.com/agritheory/shipstation_integration/issues/2
-before_migrate = "shipstation_integration.tests.setup.setup_custom_fields"
+# before_migrate = "shipstation_integration.migrate.before_migrate"
```

Comment:
We need the after migrate hook here too to ensure these fields get sync'd. I would like to use the one in test_utils but it isn't ready yet

Reply:
Wouldn't the `bench migrate` command automatically sync up everything in the `custom` folder? Should this PR wait until the test-util hook is ready?